packages/dashboard/components/consent/utils/validateSession.ts

```diff
@@ -0,0 +1,77 @@
+import { LitPKPResource } from "@lit-protocol/auth-helpers";
```

Comment:
Let's call this `getValidSessionSigs()` (and adjust the filename accordingly) for clarity